src/app/person/person.cpp

```diff
@@ -0,0 +1,43 @@
+#include "person.hpp"
+
+Person::Person(std::string firstName, std::string lastName, std::string pesel)
+{
+    setFirstName(firstName);
+    setLastName(lastName);
+    setPesel(pesel);
+}
+
+Person::~Person()
+{
+}
+
+void Person::setFirstName(std::string firstName)
```

Comment:
For setters pass arguments with const, as they shouldn't be changed 
`const std::string first_name`
Also apply snake case to all args